src/tests/test-helpers.ts

```diff
@@ -1,10 +1,19 @@
 import { expect } from "vitest";
 import { PurchasesError } from "../entities/errors";
+import { type Offerings } from "../entities/offerings";
 
 export function failTest() {
   expect(true).toBeFalsy();
 }
 
+export function assertExpectedOfferings(
```

Comment:
Now, we can't do an equals directly in the offerings, since it has a function as well...

Reply:
See previous proposal 🙂